### DISCUSSION TITLE
min browser with zoom page?

### DISCUSSION THREAD

**Discussion by ghost** [minbrowser/min#2155] [Q&A] [answered]
Hi everyone.

#### concept
I would like to know if there would be support for the [zoom-js](https://lab.hakim.se/zoom-js/) library in the Min browser to add a layer of user interaction and zoom in any web page. Please, see this [source-code](https://github.com/hakimel/zoom.js)

#### why?
1. browsers like [internet explorer](https://learn.microsoft.com/en-us/previous-versions/hh771891(v=vs.85))  offer this feature
2. the zoom feature is very useful for accessibility issues

#### syntax
```
-ms-content-zooming property
```
**code-description:** This code specifies whether zooming is enabled. this property is read/write.

#### question/feedback
*What do you all think of the idea?*

**Comment by PalmerAL** [comment, marked as answer]
We support zooming like this using a pinch gesture on trackpads already. So this would just be for regular mice, right?

I think I can see how this would be useful for people without trackpads, but I'm not sure how many people would use it. Perhaps it should be a [userscript](https://github.com/minbrowser/min/wiki/userscripts)?

I'm confused on the "syntax" section. I understand Internet Explorer had a property to let pages enable/disable this feature, but I expect very few pages today implement that. And we can't make up our own CSS extensions (because our market share is 0.00000.....001%). If we did this, I imagine it would be automatically enabled on every page?

**Reply by ghost** [reply]
Hi PalmerAL.

> We support zooming like this using a pinch gesture on trackpads already. So this would just be for regular mice, right?

true.

> I think I can see how this would be useful for people without trackpads, but I'm not sure how many people would use it. Perhaps it should be a [userscript](https://github.com/minbrowser/min/wiki/userscripts)?

great idea. ^^

> I'm confused on the "syntax" section. I understand Internet Explorer had a property to let pages enable/disable this feature, but I expect very few pages today implement that. And we can't make up our own CSS extensions (because our market share is 0.00000.....001%). If we did this, I imagine it would be automatically enabled on every page?

yes... so, thank you for feedback.
